index.js

```diff
@@ -43,6 +43,79 @@
 //.
 //. Sanctuary is designed to work in Node.js and in ES5-compatible browsers.
 //.
+//. ## Ramda
+//.
+//. Ramda provides a number of partial functions. These necessitate the use
```

Comment:
Putting "partial function" in parenthesis (possibly linking to a more precise/comprehensive definition) would provide an education opportunity as well.

Something like "Ramda provides a number of functions which may return `null` or `undefined` ([partial functions](https://en.wikipedia.org/wiki/Partial_function))".

Same thing with [totality](https://en.wikipedia.org/wiki/Partial_function#Total_function) and [variadic functions](https://en.wikipedia.org/wiki/Variadic_function).